web/src/features/moreCast2/util.ts

```diff
@@ -123,6 +127,32 @@ export const mapForecastChoiceLabels = (newRows: MoreCast2Row[], storedRows: Mor
   return newRows
 }
 
+export const fillForecastsFromRows = (rowsToFill: MoreCast2Row[], savedRows: MoreCast2Row[]): MoreCast2Row[] => {
+  for (const row of rowsToFill) {
```

Comment:
I think this could be simplified a bit:
- `if(!savedRows)` then return `rowsToFill` right away as an if guard
- Can we also add that `savedRows` is potentially `undefined` or `null` in the parameter type?
- Can rewrite the `for of` loop to a filter: `rowsToFill.filter(row => isForecastRow(row)).map(forecastRow => { // ... do the things })`